miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/analytics/Type.kt

```diff
@@ -6,7 +6,23 @@ internal enum class Etype(val value: String) {
 }
 
 internal enum class Actype(val value: String) {
+    SEND_MESSAGE_TO_CONTACT_ID("mini_app_sendMessageToContactId"),
     HOST_LAUNCH("mini_app_host_launch"),
     OPEN("mini_app_open"),
-    CLOSE("mini_app_close")
+    CLOSE("mini_app_close"),
+    GET_USER_NAME("mini_app_getUserName"),
+    SET_SCREEN_ORIENTATION("mini_app_setScreenOrientation"),
+    GET_CONTACTS("mini_app_getContacts"),
+    SEND_MESSAGE_TO_CONTACT("mini_app_sendMessageToContact"),
+    REQUEST_PERMISSION("mini_app_requestPermission"),
+    REQUEST_CUSTOM_PERMISSIONS("mini_app_requestCustomPermissions"),
+    SHARE_INFO("mini_app_shareInfo"),
+    LOAD_AD("mini_app_loadAd"),
+    SHOW_AD("mini_app_showAd"),
+    GET_POINTS("mini_app_getPoints"),
+    GET_PROFILE_PHOTO("mini_app_getProfilePhoto"),
+    GET_ACCESS_TOKEN("mini_app_getAccessToken"),
+    SEND_MESSAGE_TO_MULTIPLE_CONTACTS("mini_app_sendMessageToMultipleContacts"),
+    GET_UNIQUE_ID("mini_app_getUniqueId"),
```

Comment:
(optional feedback)

We can preserve the same order to keep those entries according to https://github.com/rakutentech/android-miniapp/blob/master/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt#L132~L150

However, `ActionType` and `Actype` enum have those duplicate entries per feature but with different constant values - If we could skip using `Actype` enum in `MessageBridgeRatDispatcher`, then we could produce those entries as following: 

```
internal class MiniAppAnalytics(
    val rasProjectId: String,
    ...
) {

    internal fun createParams(
            ...
            actype: String, // ----> as String.
            ...
        ): Map<String, Any> { ... }

    internal fun sendAnalyticsBridge(
            eType: Etype,
            actype: String,
            miniAppInfo: MiniAppInfo?
    ) = try {
        val params = createParams(
                rasProjectId = rasProjectId,
                acc = BuildConfig.ANALYTICS_ACC,
                aid = BuildConfig.ANALYTICS_AID,
                actype = actype,
                miniAppInfo = miniAppInfo
        )
        trackEvent(eType, params)
    } catch (e: Exception) {
        ....
    }
}

internal class MessageBridgeRatDispatcher(private val miniAppAnalytics: MiniAppAnalytics) {

    fun sendAnalyticsSdkFeature(action: String) {

        miniAppAnalytics.sendAnalyticsBridge(
            eType = Etype.CLICK,
            actype = "mini_app_$action",
            miniAppInfo = null
        )
    }
}

```

It will introduce few refactoring - I'm not sure if it's a good idea. Probably, we can address any improvement in another PR.

Reply:
@khairul-alam-licon i also thought about that, Bt now `actype = "mini_app_$action"` is how i implemented but if in the documentation the constant value of the actype change a little bit then my approach in the PR is better, @corycaywood what do you think?